src/components/navbar.svelte

```diff
@@ -49,11 +46,16 @@
         <div class="mdc-top-app-bar__row">
           <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
             <button on:click={() => menu.setOpen(true)} class="material-icons mdc-top-app-bar__navigation-icon mdc-icon-button" aria-label="Open navigation menu">menu</button>
-            <span class="mdc-top-app-bar__title">Hello {$players[$players.length-1].name}, Welcome to Unified Arcade App</span>
+            <span class="mdc-top-app-bar__title">Hello {$currPlayer.name}, We come to Unified Arcade App</span>
```

Comment:
this is much easier to read

Reply:
And We come is much better than welcome. :)